### ISSUE TITLE
Ganache 2.0.0 GUI not opening on Linux Mint

### ISSUE THREAD

**Issue by mzeeqazi** [trufflesuite/ganache#1196]
## Expected Behavior
GUI will open as normal

## Current Behavior

![gana](https://user-images.githubusercontent.com/16624466/55310012-0f8c2a00-547d-11e9-82df-9741d7966678.png)

Am here for last 20 minutes
## Context

I already have earlier version of Ganache(1.3.1) on my system, I uninstalled it. But now am not able to use latest version. 

## Your Environment

* Version used:2.0.0
* Operating System and version: Linux Mint 19.1

**Comment by dstreppa** [comment]
Same for me on Ubuntu LTS, no way to run it. Now I have downgraded to 1.3.1.

**Comment by davidmurdoch** [comment]
I haven't tested on Linux Mint, mzeeqazi, but I have tested the AppImage on Ubuntu 18.04.2 LTS and can't reproduce; after changing the permissions so the file can be executed (`chmod 775 Ganache-2.0.0.AppImage`). /cc @dstreppa

Is there anything out of the ordinary about your system?

**Comment by mzeeqazi** [comment]
@davidmurdoch  There is nothing extraordinary with my system. I was able to install and play with it at first. But now it's not responding. Is there any way I can get log or something.

**Comment by dstreppa** [comment]
@mzeeqazi I write here in your issue because I think our problem is the same. @davidmurdoch It's not  a permission problem, as you can see in the following output:

<pre>
dstreppa@DSTREPPA:~$ cat /etc/os-release 
NAME="Ubuntu"
VERSION="18.04.2 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.2 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic

dstreppa@DSTREPPA:~/Software/Ganache$ chmod 755 Ganache-2.0.0.AppImage 
dstreppa@DSTREPPA:~/Software/Ganache$ ll Ganache-2.0.0.AppImage 
-rwxr-xr-x 1 dstreppa dstreppa 105471886 Apr  2 15:33 Ganache-2.0.0.AppImage*
dstreppa@DSTREPPA:~/Software/Ganache$ ./Ganache-2.0.0.AppImage
</pre>

Ganache is now pending...forever and ever :thinking: . For completeness, below `ps` output, it seems Ganache processes are running somewhere in the background, but the GUI do not appear.

<pre>
dstreppa@DSTREPPA:~$ ps aux  | grep Ganache | grep -v grep
dstreppa 21002  0.2  0.6 1558408 111344 pts/0  Sl+  08:27   0:01 /tmp/.mount_GanachECSXxN/Ganache
dstreppa 21006  0.1  0.0  22000  1832 ?        Ssl  08:27   0:00 ./Ganache-2.0.0.AppImage
dstreppa 21018  0.0  0.1 338124 29872 pts/0    S+   08:27   0:00 /tmp/.mount_GanachECSXxN/Ganache --type=zygote --no-sandbox
dstreppa 21048  0.0  0.2 401900 43444 pts/0    Sl+  08:27   0:00 /tmp/.mount_GanachECSXxN/Ganache --type=gpu-process --no-sandbox --supports-dual-gpus=false --gpu-driver-bug-workarounds=1,7,24,63,76 --disable-gl-extensions=GL_KHR_blend_equation_advanced GL_KHR_blend_equation_advanced_coherent --disable-webrtc-hw-encoding --gpu-vendor-id=0x8086 --gpu-device-id=0x191b --gpu-driver-vendor --gpu-driver-version --gpu-driver-date --service-request-channel-token=D506178DF8DEB5FC8401139F0DEF76FF
dstreppa 21053  0.1  0.4 964492 73532 pts/0    S+   08:27   0:00 /tmp/.mount_GanachECSXxN/Ganache /tmp/.mount_GanachECSXxN/resources/app.asar/src/truffle-integration/index.js

</pre>

**Comment by dstreppa** [comment]
@davidmurdoch after a CTRL-C I have this output:

<pre>
dstreppa@DSTREPPA:~/Software/Ganache$ ./Ganache-2.0.0.AppImage 
^CTruffle Integration process exited prematurely due to signal 'SIGINT'.
(node:11113) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 2 watcher-stopped listeners added. Use emitter.setMaxListeners() to increase limit
</pre>

**Comment by davidmurdoch** [comment]
@dstreppa, can you back-up (so we can try to reproduce if this directory is the culprit) and then delete Ganache's config directory located at `~/.config/Ganache`?

**Comment by mzeeqazi** [comment]
@davidmurdoch deleting  `~/.config/Ganache` worked for me.

**Comment by davidmurdoch** [comment]
@mzeeqazi do you mind zipping up your `~/.config/Ganache` folder and sending it to us to use for debugging?

**Comment by mzeeqazi** [comment]
@davidmurdoch I hope I can, but I have  rm -rf that directory in excitement.

**Comment by dstreppa** [comment]
@davidmurdoch Yes, the problem was the .config/Ganache folder. I suspected there was a config file broken or something like this. I tried with `find /home/dstreppa/ -type f -name "*anach*"`, but I forgot that with `-f` parameter `find` reports only files :laughing:. However, I have zipped the folder (~290 MB) if you want. Stupid question, maybe not...where can I send it?

**Comment by davidmurdoch** [comment]
@dstreppa I think Ganache UI likely changed the format of one of the config files in a backwards incompatible manner which is causing the hang.

You can use https://send.firefox.com/ and paste the link here if you don't mind sharing the file publicly. Alternatively you can email it to me at david@trufflesuite.com
